core/conversion/converters/converter_util.cpp

```diff
@@ -151,9 +152,30 @@ nvinfer1::ILayer* add_elementwise(
     } else {
       // other is with static shape, expand dimension to make tow tensor have
       // the same number of dimension
-      auto otherShuffle = ctx->net->addShuffle(*other);
-      otherShuffle->setReshapeDimensions(util::toDimsPad(otherDim, selfDim.size()));
-      other = otherShuffle->getOutput(0);
+
+      // first check if the selfDim can be reduced to the other
+      int diff = selfDim.size() - otherDim.size();
+      bool canSqueeze = true;
+      for (int i = 0; i < diff; i++) {
+        if (selfDim[i] != 1) {
+          canSqueeze = false;
+        }
+      }
```

Comment:
Does this work if `diff < 0`?

Reply:
Yes it probably should since the above condition ensures that the self has greater dimension than the other. But this is leading to dimensions getting mishandled in bert model, so I need to removing this logic